### DISCUSSION TITLE
how can I silent the message Authenticity is not established by certificate path validation

### DISCUSSION THREAD

**Discussion by juanpabloaj** [visciang/telegram#141] [General]
Every time I do a telegram request I receive this message

```
[warning] Description: 'Authenticity is not established by certificate path validation'
     Reason: 'Option {verify, verify_peer} and cacertfile/cacerts is missing'
```

do you know how can I silent that message? 

thanks in advance.

**Comment by visciang** [comment]
Hey @juanpabloaj,
which HTTP client are you using?

See: "Telegram Client Config" under the docs https://visciang.github.io/telegram/readme.html#poller

**Reply by juanpabloaj** [reply]
thanks!

the instructions in the link solve the issue.

sorry I hadn't checked that file.
I was using the default config.

thanks again
